lesson_13/homework_lesson_13.py

```diff
@@ -0,0 +1,167 @@
+"""Завдання 1:
+
+Візміть два файли з теки ideas_for_test/work_with_csv 
+порівняйте на наявність дублікатів і приберіть їх. 
+Результат запишіть у файл result_<your_second_name>.csv"""
+
+"""Хід роботи:
+1. Перевірити наявність дублікатів в одному файлі - прибрати.
+2. Перевірити наявність дублікатів в іншому файлі - прибрати.
+3. Порівняти два файли на наявність дублікатів і винести дані з них у третій файл."""
+
+import csv
+from pathlib import Path
+import json
+import os
+import logging
+import xml.etree.ElementTree as ET
+
+# видалити дублікати, залишивши першу появу дублікату у стовпці 'ContactID' з файлу 'random-michaels.csv'
+f_csv_path_1 = Path("D:\\Courses\\Hillel IT School\\QA Automation Python\\HW_QA Python\\lessons\\les_13_hw\\random-michaels.csv")
```

Comment:
```suggestion
f_csv_path_1 = Path("D:\\Courses\\Hillel IT School\\QA Automation Python\\HW_QA Python\\lessons\\les_13_hw\\random-michaels.csv")
```
недолік такого шляху явний - на іншому компютері його не існуватиме